rslint_cli/src/lib.rs

```diff
@@ -70,6 +70,11 @@ pub fn run(glob: String, verbose: bool) {
         CstRuleStore::new().builtins()
     };
 
+    if walker.files.len() == 0 {
```

Comment:
```suggestion
    if walker.files.is_empty() {
```
Minor change like clippy said